### ISSUE TITLE
Authentication and onboarding

### ISSUE THREAD

**Issue by Jayjerome** [Jayjerome/GoFundMe#1]
Develop onboarding and authentication screens

**Comment by Collinsturi** [comment]
I have already created the  splash screen in my UI branch. Let me finish on creating the onboarding fragments then merge with the dev branch. From there we can discuss how we will store and receive the user information.

**Comment by Collinsturi** [comment]
Splash Screen, Bottom sheet  fragment, Sign In fragment and Log In fragment are complete.  They are all working  as expected except the  Bottom sheet fragment that has a design bug which I'm currently working on(It doesn't have rounded corners) . 

I haven't considered the different screen sizes in these designs but I'm hoping this will be done after finishing with the dashboard.

**Comment by Collinsturi** [comment]
![Screenshot (131)](https://user-images.githubusercontent.com/86106046/201972169-e89c2f8a-0d44-46b7-92a8-74628d56fbb6.png)

![Screenshot (133)](https://user-images.githubusercontent.com/86106046/201972245-feab6d66-f20e-492a-ad00-8410961a0058.png)


![Screenshot (134)](https://user-images.githubusercontent.com/86106046/201972273-c9a07cdc-dea4-47d5-8457-2442fb7d34fa.png)


![Screenshot (135)](https://user-images.githubusercontent.com/86106046/201972312-bc7d2536-f2a2-4bd3-b0ed-4b693931a6f8.png)


![Screenshot (136)](https://user-images.githubusercontent.com/86106046/201972429-309a1b77-1d69-4074-aadb-af1a5b63029d.png)

**Comment by Collinsturi** [comment]
The Bottom sheet needs a description

**Comment by Jayjerome** [comment]
> Splash Screen, Bottom sheet fragment, Sign In fragment and Log In fragment are complete. They are all working as expected except the Bottom sheet fragment that has a design bug which I'm currently working on(It doesn't have rounded corners) .
> 
> I haven't considered the different screen sizes in these designs but I'm hoping this will be done after finishing with the dashboard.

Okay great. You can use an xml file with top level radius as background of that view

**Comment by Jayjerome** [comment]
> The Bottom sheet needs a description

Yes, someone in the team will handle contents on the application

**Comment by Collinsturi** [comment]
> > Splash Screen, Bottom sheet fragment, Sign In fragment and Log In fragment are complete. They are all working as expected except the Bottom sheet fragment that has a design bug which I'm currently working on(It doesn't have rounded corners) .
> > I haven't considered the different screen sizes in these designs but I'm hoping this will be done after finishing with the dashboard.
> 
> Okay great. You can use an xml file with top level radius as background of that view

Already taken care of.

![Screenshot (137)](https://user-images.githubusercontent.com/86106046/202041507-f0c9dc65-06bb-4a0c-a3bb-68a6f9f60fc4.png)

**Comment by Collinsturi** [comment]
![Screenshot (142)](https://user-images.githubusercontent.com/86106046/204060837-3040a0de-a867-4168-9f1a-c61940dcd714.png)


The Verification Fragment has been added
